Add Very Strong + Magic Hint Distro

dotzo opened the pull request:
With a few more no logic races happening recently, having this distro on hand would be useful.  It's idential to the Very Strong distribution, but adds a Path of Magic hint. Will work for all logics, but no logic specifically would use this all the time.

dotzo committed:
Add Very Strong + Magic Hint Distro

DobbyDigital commented:
Have you tested it to make sure that it generates with a Path of Magic hint? "use_default_goals" is set to True, and the priority for the Path of Magic is 90, so I would anticipate that it would generate one of the other path hints first (depending on what the bridge, Ganon BK, and Trials are set to)

DobbyDigital committed:
Force Path of Magic hint

Prevents hinting any of the default goals, and adds a fixed hint for Path of Magic. I've tested it, and seems to consistently generate one Path of Magic hint with two copies. Unclear whether it should have one or two copies, since the numbers vary by type in this hint distro, but that's easy to change.

dotzo committed:
Merge pull request #1 from DobbyDigital/patch-1

Force Path of Magic hint
